lib/components/Stepper/index.tsx

```diff
@@ -0,0 +1,77 @@
+import React from "react";
+import clsx from "clsx";
+import "./stepper.scss";
+import { LabelPairedCheckMdFillIcon } from "@deriv/quill-icons";
+import { Text } from "@components/Typography";
+import { TBasicStepperProps } from "./types";
+import { VerticalStepper } from './vertical-stepper';
+import { HorizontalStepper } from './horizontal-stepper';
+import { TypographyProps } from "@components/Typography/base";
+
+const IconOk = ({ size }: { size: TypographyProps['size'] }) => (
+    <LabelPairedCheckMdFillIcon
+        fill="var(--semantic-color-slate-solid-surface-frame-low)"
+        width={size === "sm" ? 11 : size === "md" ? 13 : 14}
```

Comment:
This icon must have followed the design system size. Why do we have `13` here, can we double check with the designers?

Reply:
There no token for this size, I've requested it here: https://deriv-group.slack.com/archives/C054V5QFXD3/p1728041460209409
let's wait

Reply:
changed✅